MainModule/Server/Dependencies/TrelloAPI.lua

```diff
@@ -10,9 +10,9 @@
 -- It is requested that existing credits remain here.																											 --
 -------------------------------------------------------------------------------------------------------------------------------------------------------------------
 
-local print = function(...) for i,v in pairs({...}) do warn("[Adonis TrelloAPI]: INFO: "..tostring(v)) end end
-local error = function(...) for i,v in pairs({...}) do warn("[Adonis TrelloAPI]: ERROR: "..tostring(v)) end end
-local warn = function(...) for i,v in pairs({...}) do warn("[Adonis TrelloAPI]: WARN: "..tostring(v)) end end
+local print = function(...) for i,v in {...} do warn("[Adonis TrelloAPI]: INFO: "..tostring(v)) end end
+local error = function(...) for i,v in {...}) do warn("[Adonis TrelloAPI]: ERROR: "..tostring(v)) end end
```

Comment:
You kept a ) in here still!

Reply:
Ah, I do apologise, fill  get that fixed now! Thank you!